docs/cli/cmdline/script/info.md

```diff
@@ -0,0 +1,49 @@
+---
+title: terramate script info - Command
+description: The script info command gives you information about a script
+
+prev:
+  text: 'Run'
+  link: '/cli/cmdline/run'
+
+next:
+  text: 'Script List'
+  link: '/cli/cmdline/script/list'
+---
+
+# Script Info
+
+**Note:** This is an experimental command that is likely subject to change in the future.
+
+The `script info NAME` command lists details about all script definitions with the given NAME. The details include:
```

Comment:
```suggestion
The `script info SCRIPT-NAME...` command lists details about all script definitions with the labels matching the given SCRIPT-NAME. The details include:
```

Reply:
I've updated this to be in line with `run`, but I have chosen to link to the run page rather than duplicate the same info, since in future we might have other ways of targetting labels (e.g. groupings) and IMO it should only be defined in one place